WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfo.cs

```diff
@@ -1,7 +1,9 @@
-using ReactiveUI;
 using System;
```

Comment:
Probably this can be moved somewhere. According to our current concept, we are doing functional grouping. I am OK with not touching this file location too.

Reply:
Is this a ViewModel? TransactionInfoViewModel?